apps/explorer/src/api/assets/assetsApi.ts

```diff
@@ -72,54 +72,58 @@ const NFTCollectionsMetadata = create({
 })
 
 // Queries
-export const assetsQueries = createQueriesCollection({
+export const assetsQueries = {
   type: {
-    one: (assetId: string) => ({
-      queryKey: ['assetType', assetId],
-      queryFn: (): Promise<AssetBase> =>
-        tokensInfo.fetch(assetId).then((r) => ({ id: assetId, type: r.stdInterfaceId as AssetType })),
-      staleTime: ONE_DAY_MS
-    })
+    one: (assetId: string) =>
+      queryOptions({
+        queryKey: ['assetType', assetId],
+        queryFn: (): Promise<AssetBase> =>
+          tokensInfo.fetch(assetId).then((r) => ({ id: assetId, type: r.stdInterfaceId as AssetType })),
+        staleTime: ONE_DAY_MS
+      })
   },
   metadata: {
-    allVerifiedTokens: (network: NetworkPreset) => ({
-      queryKey: ['verifiedTokens', network],
-      queryFn: (): Promise<VerifiedFungibleTokenMetadata[]> => {
-        try {
-          return fetch(`https://raw.githubusercontent.com/alephium/token-list/master/tokens/${network}.json`).then(
-            (r) => r.json().then((j: TokenList) => j.tokens.map((v) => ({ ...v, type: 'fungible', verified: true })))
-          )
-        } catch (e) {
-          console.error(e)
-          return Promise.reject(new Error(i18n.t('Verified token fetch failed')))
-        }
-      },
-      staleTime: ONE_DAY_MS
-    }),
-    unverifiedFungibleToken: (assetId: string) => ({
-      queryKey: ['unverifiedFungibleToken', assetId],
-      queryFn: (): Promise<UnverifiedFungibleTokenMetadata> =>
-        fungibleTokensMetadata.fetch(assetId).then((r) => {
-          const parsedDecimals = r?.decimals ? parseInt(r.decimals) : 0
-
-          return {
-            ...r,
-            id: assetId,
-            type: 'fungible',
-            decimals: Number.isInteger(parsedDecimals) ? parsedDecimals : 0,
-            verified: false
+    allVerifiedTokens: (network: NetworkPreset) =>
+      queryOptions({
+        queryKey: ['verifiedTokens', network],
+        queryFn: (): Promise<VerifiedFungibleTokenMetadata[]> => {
+          try {
+            return fetch(`https://raw.githubusercontent.com/alephium/token-list/master/tokens/${network}.json`).then(
+              (r) => r.json().then((j: TokenList) => j.tokens.map((v) => ({ ...v, type: 'fungible', verified: true })))
+            )
+          } catch (e) {
+            console.error(e)
+            return Promise.reject(new Error(i18n.t('Verified token fetch failed')))
```

Comment:
I know you didn't make this change, but we shouldn't call it "Verified" anymore

Reply:
yep